timeseries/src/autogluon/timeseries/learner.py

```diff
@@ -31,13 +31,15 @@ def __init__(
         trainer_type: Type[AbstractTimeSeriesTrainer] = AutoTimeSeriesTrainer,
         eval_metric: Optional[str] = None,
         prediction_length: int = 1,
+        early_stopping_patience: Optional[int] = None,
         **kwargs,
     ):
         super().__init__(path_context=path_context, random_state=random_state)
         self.eval_metric: str = TimeSeriesEvaluator.check_get_evaluation_metric(eval_metric)
         self.trainer_type = trainer_type
         self.target = target
         self.prediction_length = prediction_length
+        self.early_stopping_patience = early_stopping_patience
```

Comment:
Let's please fold `early_stopping_patience` into `hyperparameters`. It should not be an explicit constructor / fit argument for learners or models.

Reply:
Will change it.